<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskQuestion</class>
 <widget class="QDialog" name="AskQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_money">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Money</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_money">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_love">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_love">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Love</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_money_submit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>451</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/new/prefix1/askquestion.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_love_submit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Example: Will I get a lot of money in my life?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Example: Will I find love in my life?</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton_money</zorder>
  <zorder>lineEdit_money</zorder>
  <zorder>lineEdit_love</zorder>
  <zorder>pushButton_love</zorder>
  <zorder>pushButton_money_submit</zorder>
  <zorder>pushButton_love_submit</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_money_submit</sender>
   <signal>clicked()</signal>
   <receiver>AskQuestion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_love_submit</sender>
   <signal>clicked()</signal>
   <receiver>AskQuestion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
